### ISSUE TITLE
controlnet not loading in UI

### ISSUE THREAD

**Issue by paiotao** [Mikubill/sd-webui-controlnet#364]
have done everything including the URL https://github.com/Mikubill/sd-webui-controlnet, everything is there in my SD including showing up in my settings, but no dropdown in txt2img or img2img, any ideas ???.....

Error loading script: controlnet.py
Traceback (most recent call last):
File "C:\ai\stable-diffusion-webui\modules\scripts.py", line 164, in load_scripts
module = script_loading.load_module(scriptfile.path)
File "C:\ai\stable-diffusion-webui\modules\script_loading.py", line 13, in load_module
exec(compiled, module.dict)
File "C:\ai\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 764, in
script_callbacks.on_after_component(img2img_tab_tracker.on_after_component_callback)
AttributeError: module 'modules.script_callbacks' has no attribute 'on_after_component'

Error loading script: xyz_grid_support.py
Traceback (most recent call last):
File "C:\ai\stable-diffusion-webui\modules\scripts.py", line 164, in load_scripts
module = script_loading.load_module(scriptfile.path)
File "C:\ai\stable-diffusion-webui\modules\script_loading.py", line 13, in load_module
exec(compiled, module.dict)
File "C:\ai\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\xyz_grid_support.py", line 2, in
from scripts import controlnet
File "C:\ai\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 764, in
script_callbacks.on_after_component(img2img_tab_tracker.on_after_component_callback)
AttributeError: module 'modules.script_callbacks' has no attribute 'on_after_component'

**Comment by Mikubill** [comment]
Could you share your extension version and webUI version? Maybe you need to update your WebUI

**Comment by danielraffel** [comment]
same here running version fe40f206 on macOS

**Comment by paiotao** [comment]
hey Mikubill thank you for responding, I am using sd-v1-4.ckpt (7460a6fa), not sure where to find extension version any help would be appreciated

**Comment by jason-curtis** [comment]
I had this issue when I had not run `git pull` in my `stable-diffusion-webui` in a few months!
I ran `git pull`, started up the webui and everything was working.

**Comment by JadeHMD** [comment]
I have the same problem that control net is not showing up in stable diffusion web ui ( automatic1111). I'm using all the latest versions. Could it be possible that Python is not the latest? I'm using Python 3.10.6 and that's the only thing I haven't not updated yet. But I guess this is not the reason.

I installed everything via install video tutorials. I removed control net and reinstalled it. But it doesn't help.

Thanks for help.
